packages/cli/src/zosfiles/ZosFiles.utils.ts

```diff
@@ -14,6 +14,7 @@
 import { IDataSet } from "@zowe/zos-files-for-zowe-sdk";
 
 /**
+ * @deprecated - use @zowe/zos-files-for-zowe-sdk's ZosFilesUtils.getDataSetFromName instead
  * Converts the name of a data set to an IDataSet
  * @param {string} name  - the name in the form USER.DATA.SET | USER.DATA.SET(mem1)
  */
```

Comment:
Alongside the deprecation message, should we call the `getDataSetFromName` method here?
That way, if there is ever a bug in the method, we can fix in in one place? 😋 

I don't think doing this will cause a circular dependency since the CLI package already imports the files sdk 😋

Reply:
👍

Reply:
Done